scripts/Docker/Dockerfile

```diff
@@ -0,0 +1,76 @@
+ARG BASE_IMAGE=ubuntu:20.04 
+#################################
+#   Librealsense Builder Stage  #
+#################################
+FROM $BASE_IMAGE as librealsense-builder
+
+ARG LIBRS_VERSION
+# Make sure that we have a version number of librealsense as argument
+RUN test -n "$LIBRS_VERSION"
+
+# To avoid waiting for input during package installation
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Builder dependencies installation
+RUN apt-get update \
+    && apt-get install -qq -y --no-install-recommends \
+    build-essential \
+    cmake \
+    git \
+    libssl-dev \
+    libusb-1.0-0-dev \
+    pkg-config \
+    libgtk-3-dev \
+    libglfw3-dev \
+    libgl1-mesa-dev \
+    libglu1-mesa-dev \    
+    curl \
+    python3 \
+    python3-dev \
+    ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
+
+# Download sources
+WORKDIR /usr/src
+RUN curl https://codeload.github.com/IntelRealSense/librealsense/tar.gz/refs/tags/v$LIBRS_VERSION -o librealsense.tar.gz 
```

Comment:
Why no directly from GH?

Reply:
Getting it directly from github is not possible - we are redirected to codeload. See screenshot:
![redirect_to_codeload](https://user-images.githubusercontent.com/55648466/127314607-3cccfa3c-a2ee-4931-a6f0-813bcbc9e8b8.png)